src/common/config.go

```diff
@@ -175,11 +179,14 @@ func LoadDefaults(olPath string) error {
 			Enable_seccomp:      true,
 		},
 		Trace: TraceConfig{
-			Cgroups: false,
-			Memory:  false,
-			Evictor: false,
-			Package: false,
-			Latency: false,
+			Format:        "default",
+			Log_file_dir:  logfileDir,
+			Cgroups:       false,
```

Comment:
consolidate to one setting, don't have separate Cgroups and Cgroups_level (just the latter).

Reply:
My bad, I forgot to push the latest config.json as well. This is an outdated version of the config file.